test/kata7.getEmployerRole.test.js

```diff
@@ -1,5 +1,17 @@
 const { getEmployerRole } = require("../src");
 
 describe("getEmployerRole", () => {
-  test("returns the employee's role in the company", () => {});
+  const employees = [
```

Comment:
As before, definitely need a bit more data to test here and a few more assertions, plus removal of the comment